libindy/src/services/anoncreds/prover.rs

```diff
@@ -373,7 +373,22 @@ impl Prover {
                 let attribute_value = attribute_value.parse::<i32>()
                     .to_indy(IndyErrorKind::InvalidStructure, format!("Credential attribute value \"{:?}\" is invalid", attribute_value))?;
                 Ok(attribute_value >= predicate.p_value)
-            }
+            },
+            PredicateTypes::GT => {
+                 let attribute_value = attribute_value.parse::<i32>()
+                    .to_indy(IndyErrorKind::InvalidStructure, format!("Credential attribute value \"{:?}\" is invalid", attribute_value))?;
+                 Ok(attribute_value > predicate.p_value)
+             },
+             PredicateTypes::LE => {
```

Comment:
Are we leaving off "Greater than or Equal to" PredicateType for a reason?

Reply:
This functionality isn't being exposed at the API layer, so adding this shouldn't be a reason to block this PR. It's prep work for when we decide to expose this to the API, at which point we should add tests.